app/build.gradle

```diff
@@ -102,8 +102,8 @@ dependencies {
     compile 'org.echocat.jomon:runtime:1.6.3'
 
     // kafka & zookeeper
-    compile 'org.apache.kafka:kafka-clients:2.1.0'
-    compile('org.apache.kafka:kafka_2.12:2.1.0') {
+    compile 'org.apache.kafka:kafka-clients:2.4.0'
+    compile('org.apache.kafka:kafka_2.12:2.4.0') {
```

Comment:
App doesn't need kafka, lets remove kafka dependency from here. 
Kafka is needed by api-consumption, -publishing, -misc (I don't remember all of them), but app is just a holder.